doc/troubleshooting/call-for-help.rst

```diff
@@ -72,6 +72,14 @@ It has an vibrant Spyder community as well, with new questions posted every day,
 
 .. _Stack Overflow: https://stackoverflow.com/questions/tagged/spyder
 
+.. rst-class:: fabb openteams-icon
+
+OpenTeams support
+~~~~~~~~~~~~~~~~~
+
+Get help from expert Spyder consultants on `OpenTeams`_. The OpenTeams marketplace is your one source for everything open source.
```

Comment:
```suggestion
On `OpenTeams`_, you can get help from expert Spyder consultants.
OpenTeams offers services ranging from a brief consultation to an experienced team to help you with a large scale implementation project using Spyder.
```

Conform to OSPL, match the structure of the others, give more useful and specific details about what services they provide for Spyder users (using the copy straight off their linked site) and avoid meaningless advertising taglines that don't communicate any actual information to the user (per the [Style Guide](https://github.com/spyder-ide/spyder-docs/blob/master/STYLEGUIDE.md#writing-all-languages),

> Focus on the user; this is documentation, not promotional material.

Reply:
I'm unsure about the second part suggested, I don't know if they offer services "ranging from a brief consultation to an experienced team". And I also want to note that the marketing team of open teams send us the texts they needed.

Reply:
Actually, as implied above, this is their own official copy (warts and all), as its an exact quote of what they say near the top-left of their page linked in the previous sentence:

>  Many spyder users find that they occasionally need help with installation or debugging a tricky application. **OpenTeams offers services from spyder experts ranging from a brief consultation to an experienced team to help you with a large scale implementation project.**

Including a sponsor plug is okay, but it needs to fit in an appropriate context to be effective and ethical. A pure advertising slogan like "the OpenTeams marketplace is your one source for everything open source" is fine in a dedicated section plugging the sponsor (perhaps even on a more prominent page like the index), but doesn't really make sense embedded in a section of the troubleshooting documentation listing help resources for Spyder and explaining what sort of assistance they can get at each one, whereas an except from their Snyder-specific description listing the services they provide does. This both actually gives readers useful information about what OpenTeams can offer them for troubleshooting, while also making a more compelling pitch for using their services tailored to the specifics of the context.

Reply:
Great, glad to hear that—thanks, @dalthviz